tutorials/scripting/overridable_functions.rst

```diff
@@ -12,7 +12,7 @@ This document presents the ones you'll use most often.
              notifications system. To learn more about it, see
              :ref:`doc_godot_notifications`.
 
-Two functions allow you to initialize and get nodes, besides the class's
+Two functions allow you to initialize and get nodes besides the class's
```

Comment:
I had to reread the sentence to correctly understand it here. I think there shouldn't be any comma here.

As far as I could find, you can use besides like `, besides,` and `besides,` but not `, besides`